### DISCUSSION TITLE
Einbinden von WMS-Dienst aus gView in CMS bringt Fehlermeldung

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#147] [Q&A]
Bindet man einen Service aus gView(5) als OGC Web Map Service ins CMS ein, erhält man folgende Fehlermeldung:

![WMS_aus_gView](https://github.com/gis-eni/webgis-forum/assets/100088983/f87c1c9b-89f0-440b-a9c2-9ff757fe3c8f)

Wenn man aber das GetCapabilities-Dokument (mit den Aufrufparametern REQUEST, VERSION, SERVICE) als .xml abspeichert und dieses dann im CMS einbindet, wird der Inhalt erkannt.
Kann es sein, dass der WMS Aufruf in gView ein unvollständiges GetCapabilties Dokument liefert?
https://vogis.cnv.at/gview_server/ogc/bildung_kultur@g5_geoland_rv_3857?VERSION=1.3.0&SERVICE=WMS&REQUEST=GetCapabilities

**Comment by gis-eni** [comment]
Die darfst im CMS nur den Link zum Dienst (ohne die dazugehörigen WMS Parameter REQUEST, VERSION, etc) angeben. WebGIS weiß selber, welche Parameter es für welchen Request mitgeben muss.
Manche Server liefern dann einen Fehler...

Lass die Parameter weg, dann sollte es für alle Server gehen:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/8b17f29b-5d74-490a-9d69-1e1e5f37bc96)
